recipes/freeimage/all/test_package/test_package.cpp

```diff
@@ -32,15 +34,25 @@ int main(int argc, char** argv )
     const char * image_file = argv[1];
     FREE_IMAGE_FORMAT fif = FIF_UNKNOWN;
     fif = FreeImage_GetFileType(image_file, 0);
-    if(fif == FIF_UNKNOWN) {
+    if (fif == FIF_UNKNOWN) {
         fif = FreeImage_GetFIFFromFilename(image_file);
     }
-    if((fif != FIF_UNKNOWN) && FreeImage_FIFSupportsReading(fif)) {
+    if ((fif != FIF_UNKNOWN) && FreeImage_FIFSupportsReading(fif)) {
         FIBITMAP* dib = FreeImage_Load(fif, image_file, 0);
-        if(dib)
+        if (dib)
         {
             FreeImage_Unload(dib);
         }
+        else
+        {
+            std::cerr << "Unable to load " << image_file << std::endl;
+            return 10;
+        }
+    }
+    else
+    {
+        std::cerr << "Missing plugin for " << image_file << std::endl;
+        return 10;
```

Comment:
the test_package is just there to validate that the package is correctly created, and that package consumers will be able to link against it. This is not meant to test if the library is working properly. It just needs to be able to be built and run without link errors.

Reply:
Okay, I've reverted that section.